homeassistant/components/homekit_controller.py

```diff
@@ -0,0 +1,178 @@
+"""
+Support for Homekit device discovery.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/homekit_controller/
+"""
+import http
+import json
+import logging
+import os
+import uuid
+
+from homeassistant.components.discovery import SERVICE_HOMEKIT
+from homeassistant.config import get_default_config_dir
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['homekit==0.5']
+
+homekit_dir = 'homekit'
+DOMAIN = 'homekit_controller'
+
+_CONFIGURING = {}
```

Comment:
Please use `hass.data` to store global data. It's a dictionary in which you can store anything you want.

Reply:
Done